client/my-sites/stats/stats-banners/index.jsx

```diff
@@ -135,5 +137,6 @@ export default connect( ( state, ownProps ) => {
 		isGSuiteStatsNudgeVisible: ! isGSuiteStatsNudgeDismissed( state, ownProps.siteId ),
 		isUpworkStatsNudgeVisible: ! isUpworkStatsNudgeDismissed( state, ownProps.siteId ),
 		isCustomerHomeEnabled: canCurrentUserUseCustomerHome( state, ownProps.siteId ),
+		planSlug: getSitePlanSlug( state, ownProps.siteId ),
```

Comment:
Looks like our `propTypes` have got out of sync. Should we add this and the `isCustomerHomeEnabled` one?

Reply:
Thanks for catching this! I'd rather switch to .tsx and remove propTypes all together 😁... but let's save that for another day. I'll update it.